src/modules/jcms_rest/src/Plugin/rest/resource/PingRestResource.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Drupal\jcms_rest\Plugin\rest\resource;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Provides a ping for smoke tests to call.
+ *
+ * @RestResource(
+ *   id = "ping_rest_resource",
+ *   label = @Translation("Ping rest resource"),
+ *   uri_paths = {
+ *     "canonical" = "/ping"
+ *   }
+ * )
+ */
+class PingRestResource extends AbstractRestResourceBase {
```

Comment:
There is a simpler way. But to complicate things slightly I think we should have this in it's own module.

The missing step to get this working is that we need to set the permissions to access the resource. This is typically done in the UI and then we can export the configuration.

@giorgiosironi let's discuss which is the best approach to implement this. Do we want it to always be available or is something we could enable, test, then tear down?